docs/mast/mast.rst

```diff
@@ -619,6 +655,18 @@ If no catalog is specified, the Hubble Source Catalog will be queried.
    1.7524423152919317 6382034098672634846    AIS ...              --            0
 
 
+The `~astroquery.mast.CatalogsClass.query_object` method for positional queries
```

Comment:
I think this example might be too basic for our target audience. I don't see them running into this error.